packages/bpk-component-datepicker/src/BpkDatepicker.tsx

```diff
@@ -107,7 +109,11 @@ type State = {
 };
 
 class BpkDatepicker extends Component<Props, State> {
-  inputRef: React.RefObject<HTMLInputElement>;
+  inputRef: (ref:HTMLInputElement) => void;
```

Comment:
part of me wants to make the change in BpkInput to support forwardRef... but that's a task for another day. So we're stuck with inputRef prop for now. 
I think we should rename `divRef` to `focusRef` as that's it's purpose. Then I think that should be ok 👍

Reply:
👍